daemon/daemon_unix.go

```diff
@@ -1165,6 +1165,9 @@ func (daemon *Daemon) stats(c *container.Container) (*types.StatsJSON, error) {
 	}
 	stats, err := daemon.containerd.Stats(c.ID)
 	if err != nil {
+		if strings.Contains(err.Error(), "container not found") {
```

Comment:
I guess this also needs to be done in the `_windows` version; https://github.com/moby/moby/blob/9d87e6e0fb799d6ef3bb9a97bc523f8d343b5fb3/daemon/daemon_windows.go#L525-L529